features/step_definitions/api-token.js

```diff
@@ -115,4 +115,45 @@ module.exports = function server() {
         forbiddenKeys.forEach(property =>
             Assert.notProperty(this.testToken, property));
     });
+
+    this.When(/^they change the label associated with the token$/, () =>
+        this.randomBytes(32)
+            .then((bytes) => {
+                this.newDescription = bytes.toString('base64');
+
+                return request({
+                    uri: `${this.instance}/${this.namespace}/tokens/${this.testToken.id}`,
+                    method: 'PATCH',
+                    auth: {
+                        bearer: this.jwt
+                    },
+                    body: {
+                        description: this.newDescription
+                    },
+                    json: true
+                }).then((response) => {
+                    Assert.strictEqual(response.statusCode, 200);
+
+                    this.updatedToken = response.body;
+                });
+            }));
+
+    this.Then(/^their token will have that new label$/, () => {
+        Assert.strictEqual(this.updatedToken.description, this.newDescription);
+    });
+
+    this.Then(/^the token's 'last used' property will not be updated$/, () =>
+        request({
+            uri: `${this.instance}/${this.namespace}/tokens`,
+            method: 'GET',
+            auth: {
+                bearer: this.jwt
+            }
+        }).then((response) => {
+            const lastUsed = JSON.parse(response.body)
```

Comment:
Shouldn't you be getting and saving the lastUsed property before we make the the PATCH request, instead of reading it from the body?